db/migrate/20180930211142_combine_items_in_cart.rb

```diff
@@ -0,0 +1,13 @@
+class CombineItemsInCart < ActiveRecord::Migration[5.2]
+  def change
+    Cart.all.each do |cart|
+      sums = cart.order_items.group(:product_id).sum(:quantity)
+      sums.each do |product_id, quantity| 
+        if quantity > 1
```

Comment:
before adding cart logic to update quantity instead of creating a new order_item I used it to combine them. removed since no longer needed, as there should be no duplicated order_items in the database anymore. ✓